dynamic_screenshots/images.py

```diff
@@ -0,0 +1,201 @@
+from process_images import process_consent_matching, process_get_consent, process_get_gillick_competent_consent, process_pre_screening, process_sessions_add_dates
+
+
+IMAGES = [
+    {
+        "image_name": "consent-link.png",
+        "url_extension": "consent-forms/1",
+        "screen_size": {
+            "width": 700,
+            "height": 1080
+        },
+        "further_processing": process_consent_matching
+    },
+    {
+        "image_name": "consent-response-paper.png",
+        "url_extension": "sessions/1/consents/given",
```

Comment:
As with my comment on the readme, I suggest you rename `url_extension` to `path`.